### PR TITLE
Telco Lite story

### PR DESCRIPTION
**Pull request by alexronquillo**
## Description

Adds a new set of docs for the Acme Telco Lite `demo-deployer` story.

## Reviewer Notes

- I branched off of @jerelmiller's innovation week branch to make use of some of the upcoming code block changes. **That branch needs to be merged first**.
- When you review it locally, the article is [here](http://localhost:8000/automate-workflows/diagnose-problems)

## Related Issue(s) / Ticket(s)

- [JIRA ticket](https://newrelic.atlassian.net/secure/RapidBoard.jspa?rapidView=1684&projectKey=DECODOCS&modal=detail&selectedIssue=DECODOCS-15)

### CODE REVIEW

**Comment by jerelmiller** [comment]
FYI, I temporarily shifted the base branch to my branch so that the diff is a bit smaller

File: src/markdown-pages/automate-workflows/diagnose-problems/diagnose-problems.mdx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+- [AWS Telco Lite Demo](https://raw.githubusercontent.com/newrelic/demo-deployer/main/demo/catalog/telcolite.aws.json)
+- [Azure Telco Lite Demo](https://raw.githubusercontent.com/newrelic/demo-deployer/main/demo/catalog/telcolite.azure.json)
+- [GCP Telco Lite Demo](https://raw.githubusercontent.com/newrelic/demo-deployer/main/demo/catalog/telcolite.gcp.json)
+
+Follow the [Deployement guide](https://github.com/newrelic/demo-deployer/tree/main/demo/README.md#deploy-your-services) in the deployer's repository for a thorough explanation of how to use the deployer in a local Docker environment. When you run the deployment script, make sure to pass the url you copied for `<demo-url>`.
+
+Since Telco Lite contains several services, the deployment can take over half an hour. When it finishes, you should see some output stating that the deployment was successful:
+
+```shell copy=false
```
Comment (jerelmiller) [inline src/markdown-pages/automate-workflows/diagnose-problems/diagnose-problems.mdx:62]:
```suggestion
```shell copyable=false
```

**Review by jerelmiller — COMMENTED** [review]
Small suggestion which should make sure the copy button is hidden properly

File: src/markdown-pages/automate-workflows/diagnose-problems/diagnose-problems.mdx (hunk 1 above)
Comment (alexronquillo) [reply src/markdown-pages/automate-workflows/diagnose-problems/diagnose-problems.mdx]: thanks!

**Comment by jerelmiller** [comment]
@alexronquillo When this is ready to go and gets final approval, go ahead and merge it into `develop`. I'll cut a release once this is merged.

**Review by jerelmiller — APPROVED** [review]

**Pull request merged**